src/app/(with-header)/myactivity/components/ScheduleSelectForm.tsx

```
@@ -0,0 +1,64 @@
+        <button
+          type='button'
+          onClick={onAddDate}
+          className='mt-6 bg-green-300 px-10 py-5 hover:bg-green-600'
+        >
+          <p className='text-2xl font-bold text-white'>+</p>
+        </button>
+      </div>
+
+      {dates.map((dateSlot, idx) => (
+        <div className='flex'>
+          <ScheduleSelect
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)